products/angular.md

```diff
@@ -13,7 +13,7 @@ releases:
     release: 2021-11-04
     support: 2022-05-04
     eol: 2023-11-12
```

Comment:
eol dates are wrong for 13.x series according to here : 
https://angular.io/guide/releases#support-policy-and-schedule

It says its eol is May 04, 2023

Reply:
Thanks for pointing out, updated the date.